notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb

```diff
@@ -0,0 +1,1693 @@
8pQj6SZBDg2/pKBuh4abRYuAn/zEESYAxw7d0CDbzkbQ6Myh0bdvbpnlCwuB225LdXUEYRwa7movJuSkvFwS+QEMOSG9AxsODbdDwLagkU3ISbaChqlW5a60NHFialnuIIeE9zgcOyZ96siRjgBh+inj0ACAuXOl7+7XTxwaXoYNE6H2qacy7x9Id2gY957boWGIMoeGOQ5GwKGgQcipSbaChrkPd0QpNQpAG4CJ0TSJEEIcWludQYlNhwYQ34DZCAK5OjRGjJAkm26Hhkm6GSZBDg0zaDV5M4B0h0YmBg2Sgacp2ZrJiuy9G9rRIfs34sHYsU7ZwUwlaLtDSYmIGe5SjGbgPGiQ5CwB6NAgvQMbOTTcVSNsCxqFhfKXyaGRbQ4NU+HELWiMHy/9qdl+JoeGuw1mn5kcGoAT3nLttU4f7+WGGySPx4ED3Q85AaQPjap/dJ+PFDQIObXJVtB4Qik1CMBPAbwFYCeAP0bVKEIIMZiBEGBP0IjboWEGoNnm0NDaETTGjxdBQ+voBI1MDg1TIcV8h9paqVRiBrKZMA6N6mp5nkvISX29iBrmmBUUyGQgGyGlq/j9FocPi4AyYIAjaETZBkLyhbgFBW8OibhzaPjlZPAL9wAch0ZjY3bXsSBBA3AE36CQD+/vYERlr0OjoyNVlDn3XFl++MPB7brhBvncn/8s/X93BY0xY6SvjgI/h4atsq2EkGjJmBRUKXU2gCqt9Y+Sz/sDWAtgE4BfRN88Qkhvx9z1B+IPOWlrk9jbuB0auebQaG6WQXJFhYgHR47IcauqAt7//vDbl8mhcfrpwNq1ImRMniyD6YqK7Aat5eVS5tQM2DMJGt7QG78wndtuc94XBe42mLKsDQ1yvhhB5fHHnTwhhJzK9OmTLnJGSU2N9DcG2w4NQPqETA4NQMTeCRMyb3vnTnF7uPtAI2js3i35eTIlBe3okOtXUZET5jJihONoaWiQNnV0OCLDeedJ0uaZM4PbNXu2hPP99rfyPNsqJ4cOidBrrqVG0Igq3ARIFbjo0CDk1KazIeZ/AzgBAEqpCwH8JLmuEcDd0TaNEELsOjTMXbW4HRqbNjmJ2rz45dBwD1jNoHf5cnlPXA6NI0dk8DhrVmqbamuzT8rpdmgUFmZ2NniFHT9B473v9Y8FDws/cckIGoZrruFdQdI7iDPkROv00s5h988rVkgOorffTn+tvV0EBe//dlAbzLUEyC6Pxo4d0pcXuW47ugWN9vZUsd3bBsBph/v64HZomGur+ztkEjMAESVuuME5Jrk4NMrLnQSuo0bJMkpBw/1bUNAg5NSmM0GjUGtdn3z8QQB3a60f1lp/B8CUaJtGCCGpDo24BQ13PgQgHkGjowN4+mngssv8X/ebRLtjpM2g9+WXZRlXDg0zODYDYtOmfftyEzQOH5aJyqhRzuDXD29SUCNoxJmvwk9ccocoEdKbiNMh0dQkeXqiEjSOHQM+9jHpvx55JP11k1TTT9AIKttqXGrZ5NHYsUOSgLoZM0YEhd27HcdHkEMDcI7Fvn3yueHDnWtZQ4M4TIDswgHd3HCD87gzQcNd5cS9n+HDpZ+cMSO3fecCHRqE9B46FTSUUkYfvhTAS67XMoarEEJIGLgdGnGHnLgrVgDxDNjfeksm50HOAj9Bw8+h8corsozLoWGEpxkzZOBu2rRvX/Y5JMrL5U7m7t2d37nzJgXNVBkmKoIcGhw0k95InIKGX1haprKxJolltnznO8DGjSKQvvRS+utGDMgl5MSEmWTr0HDnzzDbHjVK+kdzbfJzaHj7xtpa+R5FRfLXv3+wQyMb5s51xJYgQcOIFyZB9aFDUlLcUFAArFsHfO1rue07F9znY329fHcTGkgIObXoTNB4AMASpdSjkEonLwOAUmoKJOyEEEIipbc5NJ58Uu6mXXGF/+uZQk4qKmRwWlYGvPmmrIvboTF8uAyea2slhvvgwewdGoMGiZV8/frMFU4A/5ATpXK/29gd/H4Lb8gJIb2FTIJC2PhVQsoU8nLddcCXv5zdtl97DfjZz4BbbgE++UkJ3zNVmwxBd/wzhZyYUtKdCRrNzdJvegUNQESR3bsdcSabkBOvS86E9nXVoWHCToBgQWPMGGDaNODZZ+V5XV36fkaNSi3bHTZeh0Z5eXQVrwghdskoaGitbwPwNQC/B3C+1lq7PvfFaJtGCCEyUe7fXx7HLWj4OTT87MRdRWvgIx8BHnvMWffkk8A556TezXIT5NAoLJQBo1Li0jhxQo5bFJPr0lLZj9uh4c5iP2KEtOnAAfmOuTg0ABn8ZuvQcIecDB2aOUwlbBhyQohDnDk0/BwapaXAyZMStudl507JTZQNn/mMCMH/8R/AJZeIMGscb4Ygh0aQoNHUJP3TkCGdh5xs3y5Lb8gJ4FSxMtembEJOTAUsw6BB3XNoAMDnPy8hOZlyblx5JZBIyHXbG3ISB94cGuyXCTl16TTvvNb6Da31I1rrVte6LVprnzRJhBASLnV1MhgrKIg/5MTr0AiyE3eV3buB//1fuRPY1CQDzxUrMieyDMqh4a4kYsJOxo2L5o6UUhJ24haYjJNm6FD5vfbvd+5E5pJDw9CZoOHn0Igz3MSvDQAdGqT3EnfISUGBk1zS7B9IF521FgHClIPOREODhEJ87nPAaacBixdLKWpv2EkmQSOobOuAAY7Ym4klS2R59tnpr40fL9/d7D8Mh0ZXJvoTJgD33efkyvDjiivkmr10qR1Bo08fEaM+8Qngr3/1rxpGCDk1iKj6MyGEhIOJvfVOoOMg6hway5fLcv9+4Ec/kmSggFTGCCIo5MR9B84taERF377+Do3Bg2XgWFsrA2mga4JGtiEnbodG3IKG97c4dkz+eCeQ9EbCCDl5/nng2992xOQgqqulX3FXAfETGAEJ4Whrc8LgMrF+vSxnz5ZlWRlw7rnZCxqZcmgMGOD0jZl46ikpR+sXcjJ+vHyHjRvleWcOjY4Oub4EOTQGDUo9hmFy
```

Comment:
shuffle = false

Reply:
What's breaking sequence information if one sequence only 3 data?

Reply:
As per our discussion, resolving this